apps/console/src/features/branding/components/advanced/advance-form.tsx

```diff
@@ -155,11 +156,7 @@ export const AdvanceForm: FunctionComponent<AdvanceFormPropsInterface> = forward
         // Use a regex to replace {{lang}}, {{country}}, and {{locale}} placeholders while preserving other characters
         moderatedValue = value?.trim().replace(new RegExp(placeholdersPattern, "g"), "");
 
-        // Gracefully handle scripting attempts.
-        // TODO: This should be moved to a common util or validation library.
-        const javascriptPattern: string = "javascript:";
-
-        if (moderatedValue?.includes(javascriptPattern) || !FormValidation.url(moderatedValue)) {
+        if (URLUtils.isURLValid(moderatedValue) || !FormValidation.url(moderatedValue)) {
```

Comment:
Isn't the following the correct one?

```suggestion
        if (!URLUtils.isURLValid(moderatedValue) || !FormValidation.url(moderatedValue)) {
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5611/commits/72ff1d82a3026daad4df90f5686292b721900dbd